sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -259,6 +239,12 @@ RetT *getOrBuild(KernelProgramCache &KPCache, KeyT &&CacheKey,
 
 static bool isDeviceBinaryTypeSupported(const context &C,
                                         RT::PiDeviceBinaryType Format) {
+  backend CBackend = (detail::getSyclObjImpl(C)->getPlugin()).getBackend();
```

Comment:
Just curious. What `C` stands for in `CBackend`?

Reply:
Meant as C from Context, "ContextBackend" would be better but probably too long to type...

Reply:
Renamed it to ContextBackend in aedaacca2a830cfa69a812041f80f7887839af43